app-models/articles-models.js

```diff
@@ -59,3 +60,28 @@ exports.fetchCommentsByArticleId = (article_id) => {
       return result.rows;
     });
 };
+
+exports.insertCommentByArticleId = (article_id, username, body) => {
+  if (!username || !body) {
+    return Promise.reject({
+      status: 400,
+      msg: "missing input",
+    });
+  }
+
+  const formatInput = format(
+    "INSERT INTO comments (body, votes, author, article_id) VALUES (%L, %L, %L, %L) RETURNING *;",
+    body,
+    0,
+    username,
+    article_id
+  );
+
+  return db.query(formatInput).then((result) => {
+    // if (result.rows.length === 0) {
+    //   return Promise.reject({ status: 404, msg: "not found" });
+    // }
```

Comment:
As you noticed and commented out, this is not needed for this one endpoint. Every error case is taken care of elsewhere